### DISCUSSION TITLE
TagGroup : Even though the disabledKeys prop is already set, the onRemove event still works.

### DISCUSSION THREAD

**Discussion by lanhanh259** [adobe/react-spectrum#6682] [Q&A]
"Why does the onRemove event still work on the disabled keys, even though the disabledKeys prop is already set on the TagGroup?"

**Comment by snowystinger** [comment]
Looks like we missed propagating isDisabled down to the remove button, I've fixed it here https://github.com/adobe/react-spectrum/pull/6690
